doc/community-build/docker/cortx-all/README.md

```diff
@@ -3,6 +3,18 @@
 This document provides step-by-step instructions to build CORTX binaries and container images.
 
 ## Prerequisites
+- Please create Linux VM with CentOS 7.9 or Rocky Linux 8.4 with below minium requirements.
+    - RAM: 8GB
+    - Processor: 4
+
+- Please ensure following minimum disk space is available. 
+    - 70GB : Docker Home Directroy. Default docker home directory is /var/lib/docker. You can modify it based on disk layout. 
```

Comment:
rephrase as,

- Ensure following minimum disk space are available:
    - 70GB : Docker Home Directory where default docker home directory is /var/lib/docker. You can modify the file based on disk layout.
    - 30GB : /mnt . Use this directory as part of build process. You can use `/mnt` from system disk if you have enough disk space available.

Reply:
Fixed please review